stories/Folder-Structure.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta } from '@storybook/addon-docs';
+import { H1 } from '@leafygreen-ui/typography';
+import BannerImage from '../.storybook/static/leafygreen-ui-banner.png';
+
+<Meta title="Developer Guide/Package Folder Structure" />
+
+### Folder Structure
```

Comment:
indicate that even packages that only export one directory have a subcomponent directory i.e. button/Button/Button.tsx

Reply:
Added!